tests/Integration/goldens/dataproc/samples/V1/AutoscalingPolicyServiceClient/create_autoscaling_policy.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was automatically generated - do not edit!
+ */
+
+require_once __DIR__ . '/../../../vendor/autoload.php';
+
+// [START dataproc_v1_generated_AutoscalingPolicyService_CreateAutoscalingPolicy_sync]
+use Google\ApiCore\ApiException;
+use Google\Cloud\Dataproc\V1\AutoscalingPolicy;
+use Google\Cloud\Dataproc\V1\AutoscalingPolicyServiceClient;
+use Google\Cloud\Dataproc\V1\BasicAutoscalingAlgorithm;
+use Google\Cloud\Dataproc\V1\BasicYarnAutoscalingConfig;
+use Google\Cloud\Dataproc\V1\InstanceGroupAutoscalingPolicyConfig;
+use Google\Protobuf\Duration;
+
+/**
+ * Creates new autoscaling policy.
+ *
+ * @param string $formattedParent                               The "resource name" of the region or location, as described in
+ *                                                              https://cloud.google.com/apis/design/resource_names. * For
```

Comment:
Is the spacing here correct? Looks like there is a `* For` which might need to be on a new line?

Reply:
Good catch. It seems like these should be bullets [see the autoscaling_policy.proto](https://github.com/googleapis/googleapis/blob/master/google/cloud/dataproc/v1/autoscaling_policies.proto#L261-L270)

Reply:
Good eye Noah. I think I know exactly what caused this - running the docblock through `PhpDoc::text` to help format the line with the addition of the message to take a look at the formatting method. Taking a look at options now.

Reply:
Should be resolved now 👍